package.json

```diff
@@ -15,8 +15,8 @@
   "scripts": {
     "install:deps": "./scripts/install-deps",
     "install:validator": "./scripts/install-validator",
-    "update-abi": "yarn go:generate && yarn workspace arb-provider-ethers update-abi",
-    "build": "yarn workspace arb-provider-ethers build",
+    "update-abi": "yarn go:generate && yarn workspace arb-ts update-abi && arb-provider-ethers update-abi",
```

Comment:
Should we rename to `update:abi` in order to keep consistent with other script names?

Reply:
I think that'd be good